### ISSUE TITLE
github_repository_webhook "name" parameter can only have value "web", add to docs

### ISSUE THREAD

**Issue by sfdc-afraley** [terraform-providers/terraform-provider-github#197]
### Terraform Version
- Terraform v0.11.11
- provider.github v1.3.0

### Affected Resource(s)
- github_repository_webhook

### Terraform Configuration Files
```hcl
resource "github_repository_webhook" "hook" {
  repository = "${var.repo}"

  name = "web"  # This can only ever be web, don't change it :)

  configuration {
    url          = "${var.hook_url}"
    content_type = "json"
    insecure_ssl = false
  }

  active = true

  events = ["push", "pull_request"]
}
```
### Expected Behavior
The documentation should be updated to indicate that Github does not currently support any value other than "web" for the "name" parameter.  I've lost hours troubleshooting this in the past, and almost just did again, but luckily I remembered this issue.  At first glance this value seems arbitrary, thus the urge to change "web" to something meaningful.

**Comment by hshafy** [comment]
👍  I spent sometime too trying to figure out the problem, as per GitHub API v3 docs https://developer.github.com/v3/repos/hooks/#parameters

> Use web to create a webhook. Default: web. This parameter only accepts the value web.

Also the docs links is broken, showing this message:

`{
  "message": "As of January 2019, GitHub Services can no longer be modified. Please see the blog post for details: https://developer.github.com/changes/2018-04-25-github-services-deprecation\nYou can use the \"Replacing GitHub Services\" guide to help you update your services to webhooks: https://developer.github.com/v3/guides/replacing-github-services\n",
  "documentation_url": "https://developer.github.com/v3"
}`

**Comment by hshafy** [comment]
I see this is fixed in #183 and currently in `master` but not in the `stable-website` branch, that's why website is not updated.

**Comment by kzw** [comment]
The latest documentation does not even list `name` as one of the attributes required in this resource.  And yet it does not work without this `name = "web"`
